system-tests/project-fixtures/angular/src/app/zonejs-mocha-only.cy.ts

```diff
@@ -48,19 +47,19 @@ describe('only', () => {
 
   context.only('02 - validations', () => {
     const verifyNotPresent = (title: string) => {
-      cy.wrap(Cypress.$(window.top!.document.body)).within(() =>
-        cy
-          .contains(title)
-          .should('not.exist')
-      )
+      cy.wrap(Cypress.$(window.top!.document.body)).within(() => {
+        return cy
```

Comment:
Don't need the `return`, even though it keeps it the same as before the curly braces were added. I assume this was done automatically.